### ISSUE TITLE
loss or  avg loss cannot go below 2

### ISSUE THREAD

**Issue by lvyanhui** [pjreddie/darknet#355]
Thanks for everyone!
   I have a question: when i use yolo-voc.cfg to train for only one class "person", the loss or avg loss  go down from 270 to 2, and then hold the number at about 2？
  I have seen some articles which says the loss shall go down below 0.06.  but i cannot get the value, why?

Region Avg IOU: 0.621744, Class: 1.000000, Obj: 0.272550, No Obj: 0.004791, Avg Recall: 1.000000,  count: 3
Region Avg IOU: 0.532260, Class: 1.000000, Obj: 0.336434, No Obj: 0.005373, Avg Recall: 0.625000,  count: 8
Region Avg IOU: 0.648469, Class: 1.000000, Obj: 0.381343, No Obj: 0.003275, Avg Recall: 1.000000,  count: 1
Region Avg IOU: 0.720466, Class: 1.000000, Obj: 0.306338, No Obj: 0.006228, Avg Recall: 1.000000,  count: 3
Region Avg IOU: 0.485763, Class: 1.000000, Obj: 0.226649, No Obj: 0.004258, Avg Recall: 0.400000,  count: 5
Region Avg IOU: 0.646999, Class: 1.000000, Obj: 0.316074, No Obj: 0.004090, Avg Recall: 1.000000,  count: 2
Region Avg IOU: 0.531084, Class: 1.000000, Obj: 0.248297, No Obj: 0.003676, Avg Recall: 0.666667,  count: 6
Region Avg IOU: 0.736588, Class: 1.000000, Obj: 0.165700, No Obj: 0.004386, Avg Recall: 1.000000,  count: 3
1501: 2.276698, 2.276698 avg, 0.001000 rate, 1.371942 seconds, 96064 images
Loaded: 0.000026 seconds

**Comment by SteveIb** [comment]
Hi, did it work with you?
my avg loss is going up and down between 2-6.
any tip?I lowered learning rate from learning_rate=0.0001
to learning_rate=0.00001
